mirrord/cli/src/operator.rs

```diff
@@ -126,11 +126,12 @@ Operator License
 
     let mut sessions = Table::new();
 
-    sessions.add_row(row!["Target", "User", "Session Duration"]);
+    sessions.add_row(row!["Session ID", "Target", "User", "Session Duration"]);
 
     if let Some(status) = mirrord_status.status {
         for session in &status.sessions {
             sessions.add_row(row![
+                session.id.clone().unwrap_or_default(),
```

Comment:
```suggestion
                session.id.as_ref().unwrap_or(""),
```
Nitpick

Reply:
Need to use `as_deref` because otherwise `unwrap_or` expects `&String` instead of `&str` =]